tests/gen_many_js_functions.py

```diff
@@ -0,0 +1,40 @@
+import sys
+
+NUM_FUNCS_TO_GENERATE = 1000
+
+def func_name(i):
+	return 'thisIsAFunctionWithVeryLongFunctionNameThatWouldBeGreatToBeMinifiedWhenImportingToAsmJsOrWasmSideCodeToCallOtherwiseCodeSizesWillBeLargeAndNetworkTransfersBecomeVerySlowThatUsersWillGoAwayAndVisitSomeOtherSiteInsteadAndThenWebAssemblyDeveloperIsSadOrEvenWorseNobodyNoticesButInternetPipesWillGetMoreCongestedWhichContributesToGlobalWarmingAndThenEveryoneElseWillBeSadAsWellEspeciallyThePolarBearsAndPenguinsJustThinkAboutThePenguins' + str(i+1)
```

Comment:
Emscripten uses 2 space indentation for python files.

Reply:
It not clear to me why the test relies on long symbol names, or a large number of symbols.   Doesn't minification effect all symbols, no matter how long, or how numerous?   Wouldn't a much small handwritten test be fine?

Reply:
Large number of functions with long names is so that it will be obvious when minification occurs, i.e. one can just look at plain generated size of the program to see if minification happened or not.